### ISSUE TITLE
Outdated links in Understanding SC 2.4.2 (Page Titled)

### ISSUE THREAD

**Issue by cstrobbe** [w3c/wcag#825]
Two links in [Understanding SC 2.4.2 (Page Titled)](https://www.w3.org/WAI/WCAG21/Understanding/page-titled.html) are outdated:

* The article "Writing Better Web Page Titles" on SocialPatterns.com is no longer available; there is a newer one with a very similar title, [Writing Better Page Titles for SEO](https://socialpatterns.com/writing-better-titles/), dated 5 February 2017. I don't know to what extent the content is similar to the old article. There are several alternative articles and blog posts that are less focused on SEO, e.g. [Page titles – accessibility for web writers](https://4syllables.com.au/articles/accessibility-writers-titles/) (July 2011) and [How to write better page titles](https://www.nomensa.com/blog/2013/how-to-write-better-page-titles/) (June 2013).
* The URL http://redish.net/images/stories/PDF/interactions.html does not work either, but a PDF version of the article is available at https://redish.net/wp-content/uploads/Theorfanos_Redish_InteractionsPaperAuthorsVer.pdf.

**Comment by cstrobbe** [comment]
I am closing this issue **in protest against the [W3C's allowing overlay vendors to set up a community group](https://adrianroselli.com/2022/09/accessibility-at-the-edge-w3c-cg-is-an-overlay-smoke-screen.html)**. Overlay vendors have been known to [make false claims about their tools](https://overlayfalseclaims.com/), their products can actually [make the web worse for people with disabilities](https://www.accessibility.org.au/accessibility-overlays-make-the-web-worse-not-better-new-york-times/), and screen reader users have even created a [Google Chrome extension to block overlays](https://www.accessibyebye.org/). My contributions to WCAG will remain on halt for as long as there is a W3C community group run by overlay vendors.

**Comment by patrickhlauke** [comment]
"allowing"? i'm fairly sure W3C isn't an arbiter here, and a big part of the whole W3C process is pay-to-play when it comes to membership in the first place...

**Comment by cstrobbe** [comment]
@patrickhlauke Recommended reading: [Neutrality isn’t an option when it comes to accessibility](https://karlgroves.com/neutrality-isnt-an-option-when-it-comes-to-accessibility/) by Karl Groves. It's not about the W3C, but I'm sure I don't need to explain how it is relevant.

**Comment by patrickhlauke** [comment]
well, W3C would have to change the whole way they're set up in terms of who can become a member (currently: whoever can pay) and who can set up a community group (currently: whoever can rustle up a certain number of people to participate).

personally, i'm wary of certain state-sponsored organisations from less-than-wholesome countries being W3C members, and a large chunk of web3/crypto organisations on here...but again, they're within their rights to be here.

anyway...

**Comment by cstrobbe** [comment]
> personally, i'm wary of certain state-sponsored organisations from less-than-wholesome countries being W3C members, and a large chunk of web3/crypto organisations on here.

Recommended reading: [Whataboutism](https://fallacyinlogic.com/whataboutism/).
